test/libsolidity/syntaxTests/unexpected.sol

```diff
@@ -1,3 +1,3 @@
 unexpected
 // ----
-// ParserError 7858: (0-10): Expected pragma, import directive or contract/interface/library/struct/enum/function definition.
+// ParserError 2314: (11-11): Expected identifier but got end of source
```

Comment:
The error was better before

Reply:
I know, I tried to improve it but it's rather difficult. If we allow `a.b.c.EnumName constant variableName = a.b.c.EnumName.EnumMember`, then this can always happen. I can add a "hack" that at least detects the case of `<identiifer> <EOS>`.

Reply:
If it's a hack or too much work then I think it's fine like this. These cases are [hopefully] rare anyway

Reply:
Updated it, it's not too bad.

Reply:
Ok!